src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -387,7 +387,11 @@ dnnl::post_ops program_node::try_optimize_post_ops(dnnl::post_ops& p_ops, const
                 float scale;
                 dnnl::memory::data_type data_type;
                 cur_p_ops.get_params_sum(idx, scale, data_type);
-                new_p_ops.append_sum(scale, data_type);
+                if (is_type<convolution>()) {
+                    new_p_ops.append_sum(scale, data_type);
+                } else {
+                    new_p_ops.append_sum(scale);
+                }
```

Comment:
Could you add a brief description of this WA? such as..

// Only conv supports data type specification in append_sum. Other primitives(deconv, fc) do not support it.